### PR TITLE
Better UX for transition.

### PR DESCRIPTION
**Pull request by ar4s**
Main changes:
- add message about current transition on object detail
- change style on current transition tab

### CODE REVIEW

**Comment by ar4s** [comment]
Message on detail view:
selection_008

Current transition view:
selection_009

File: src/ralph/data_center/admin.py [hunk 1]
```diff
@@ -266,6 +267,7 @@ class DataCenterAssetSecurityInfo(SecurityInfo):
 
 @register(DataCenterAsset)
 class DataCenterAssetAdmin(
+    ActiveDeploymentMessageMixin,
```
Comment (mkurek) [inline src/ralph/data_center/admin.py:270]: mix it in VirtualServer as well

File: contrib/dhcp_agent/dhcp_agent.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def get(self, key, prefix=''):
         return last
 
     def set(self, key, value, prefix=''):
-        url_hash = get_url_hash(value)
+        url_hash = get_url_hash(key)
```
Comment (mkurek) [inline contrib/dhcp_agent/dhcp_agent.py:70]: sth is messed up here (these changes are already merged).

File: src/ralph/lib/permissions/models.py [hunk 3]
```diff
@@ -333,10 +334,10 @@ def create_permissions(
             )
             perms = Permission.objects.using(using).filter(
                 content_type=concrete_ctype,
-                codename__startswith=klass._meta.model_name
+                codename__endswith=klass._meta.model_name
             )
             if perms:
-                perms.update(content_type=ctypes)
+                perms.update(content_type=ctype)
```
Comment (mkurek) [inline src/ralph/lib/permissions/models.py:340]: same here - it was already merged

File: src/ralph/data_center/admin.py (hunk 1 above)
Comment (ar4s) [inline src/ralph/data_center/admin.py:270]: done

**Comment by mkurek** [comment]
:ok:

File: src/ralph/admin/templates/admin/base.html [hunk 4]
```diff
@@ -60,10 +60,7 @@
       <div class="row">
         <div class="small-12 columns">
           {% for message in messages %}
-            <div data-alert class="alert-box radius {{ message.tags }}">
-              {{ message }}
-              <a href="#" class="close">&times;</a>
-            </div>
+            {% include "admin/partials/message.html" with message=message tags=message.tags %}
```
Comment (szok) [inline src/ralph/admin/templates/admin/base.html]: why include,maybe better template_tag which has a cache template?

File: src/ralph/lib/transitions/templatetags/transitions_tags.py [hunk 5]
```diff
@@ -30,3 +30,37 @@ def choice_str(obj, field_name):
...
+        self.var_name = var_name
+
+    def render(self, context):
+        job = context[self.job]
+        obj = context[self.obj]
+        history = job.params['history_kwargs']
+        context[self.var_name] = history[obj.pk].items()
+        if not context[self.var_name]:
+            return '&ndash;'
+        return ''
```
Comment (ghost) [inline src/ralph/lib/transitions/templatetags/transitions_tags.py:48]: is this ok?

**Comment by ghost** [comment]
:ok:

**Pull request merged**